doc/auth/saml.md

```diff
@@ -0,0 +1,152 @@
+## Overview
+
+This guide will walk through testing Pachyderm's experimental SAML support.
+These features aren't integrated into mainline Pachyderm yet and aren't
+available in any official releases. This will describe the process of:
+
```

Comment:
Remove disclaimers from this before merging into master, since once we do that it's going out in the next official release.

Reply:
I'll do this, but I ended up pulling the docs stuff into a separate PR so they don't block submitting this